elements/pfe-jump-links/src/pfe-jump-links-nav.scss

```diff
@@ -23,39 +23,53 @@ $LOCAL-VARIABLES: (
 
 :host {
   --pfe-jump-links-panel__section--spacer: calc(var(--pfe-jump-links-nav--Height--actual) + var(--pfe-navigation--Height--actual) + var(--jump-links-nav--nudge));
-  --pfe-jump-links-panel--offset: pfe-strip-unit(#{var(--pfe-jump-links-panel__section--spacer)});
+  
+  // @TODO: This doesn't work; we can't run a Sass function on a var because a var isn't a Sass variable with a value until the var is interpolated in the DOM
```

Comment:
I'm not sure I understand this comment, where is there a sass function? does this refer to the code that was removed?
```
  --pfe-jump-links-panel--offset: pfe-strip-unit(#{var(--pfe-jump-links-panel__section--spacer)});
```

Reply:
It's commented out below it ↓